native/src/hhRoutePlanner.cpp

```diff
@@ -326,11 +327,11 @@ HHNetworkRouteRes * HHRoutePlanner::runRouting(int startX, int startY, int endX,
     if (hctx->config->ROUTE_ALL_SEGMENTS/* && route.detailed != null*/) {
         route->detailed = prepareResult(hctx->rctx, route->detailed);
     }
-    OsmAnd::LogPrintf(OsmAnd::LogSeverityLevel::Info, "%.2f ms\n", timer.GetElapsedMs() - time);
+    OsmAnd::LogPrintf(OsmAnd::LogSeverityLevel::Info, "%.2f ms\n", timer.GetElapsedMs());
     printResults(hctx->rctx, startX, startY, endX, endY, route->detailed);
     OsmAnd::LogPrintf(OsmAnd::LogSeverityLevel::Info,
-                      "Routing finished all %.1f ms: last mile %.1f ms, load data %.1f ms (%,d edges), routing %.1f ms (queue  - %.1f add ms + %.1f poll ms), prep result %.1f ms\n",
-                      timer.GetElapsedMs(), hctx->stats.searchPointsTime, hctx->stats.loadEdgesTime + hctx->stats.loadPointsTime,
+                      "Routing finished all %.f ms: last mile %.f ms, load data %.f ms (%d edges), routing %.f ms (queue - %.f add ms + %.f poll ms), prep result %.f ms\n",
```

Comment:
Why need to show not formatted floats? Dow you want to see like this:
5*-10e6
or
5.00000000099
?

Reply:
%.f cuts fractional part which is actually not needed because all timer values cames from **int** timers.